### DISCUSSION TITLE
How do I use camera continuously?

### DISCUSSION THREAD

**Discussion by Applizone** [react-native-image-picker/react-native-image-picker#1987] [Q&A] [answered]
My user will require to take large amount of photo using one hand operation. By default, after capturing photo, the screen will show confirmation (Retry, OK) and the callback will proceed to define.

How to skip this process?

My desire is after I capture the image, I can proceed with the asset object, transferring to public folder so user can continue to capture photo without need to tap OK/Retry. The user feedback they will take roughly 50 images for each task.

My current script:

```
    const captureImage = async (type) => {
        let options = {
            mediaType: type,
            quality: 1,
            videoQuality: 'high',
            saveToPhotos: true,
        };
        let isCameraPermitted = await requestCameraPermission();
        let isStoragePermitted = await requestExternalWritePermission();
        if (isCameraPermitted && isStoragePermitted) {
            launchCamera(options, (response) => {
                if (response.didCancel) {
                    return;
                } else if (response.errorCode == 'camera_unavailable') {
                    alert('Camera not available on device');
                    return;
                } else if (response.errorCode == 'permission') {
                    alert('Permission not satisfied');
                    return;
                } else if (response.errorCode == 'others') {
                    alert(response.errorMessage);
                    return;
                }
                if (Object.keys(response.assets).length > 0) {
                    var cameraResponse = response.assets[0];
                    HandleCapturePhoto(cameraResponse)
                        .then((photodata) => {
                            addPhotoToDB(photodata);
                        })
                }
            });
        }
    };
```

**Comment by Johan-dutoit** [comment, marked as answer]
I'd avoid using image-picker for this.  react-native-camera/react-native-vision-camera would be better alternatives.

**Reply by Applizone** [reply]
Yes, I've currently switch to react-native-camera and it work as expected. 

I've try react-native-vision-camera but it give me headache to settle down all the error build. I believe this library(react-native-image-picker) only applicable for non-continued task because it call camera app with all functions ready like stock camera.
